core/src/main/java/greencity/repository/RatingStatisticsRepo.java

```diff
@@ -19,4 +23,21 @@
     @Query(nativeQuery = true,
         value = "DELETE FROM rating_statistics WHERE create_date + interval '2 year' < current_date")
     void scheduledDeleteOlderThan();
+
+    /**
+     * Find {@link RatingStatistics} by page.
+     *
+     * @param pageable configuration.
+     * @return {@link Page}
+     * @author Dovganyuk Taras
+     */
+    Page<RatingStatistics> findAll(Pageable pageable);
+
+    /**
+     * Find all {@link RatingStatistics}.
+     *
+     * @return {@link RatingStatistics};
+     * @author Dovganyuk Taras
+     */
+    List<RatingStatistics> findAll();
```

Comment:
Method `findAll()` is already available in `JpaRepository` that this interface extends, so no need to introduce it here.